packages/firebase_remote_config/ios/Classes/FirebaseRemoteConfigPlugin.m

```diff
@@ -0,0 +1,176 @@
+#import "FirebaseRemoteConfigPlugin.h"
+
+#import <Firebase/Firebase.h>
+
+@interface FirebaseRemoteConfigPlugin ()
+@property(nonatomic, retain) FlutterMethodChannel *channel;
+@end
+
+@implementation FirebaseRemoteConfigPlugin
+
+static NSString *DEFAULT_KEYS = @"default_keys";
+
++ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
+  FlutterMethodChannel *channel =
+      [FlutterMethodChannel methodChannelWithName:@"plugins.flutter.io/firebase_remote_config"
+                                  binaryMessenger:[registrar messenger]];
+  FirebaseRemoteConfigPlugin *instance = [[FirebaseRemoteConfigPlugin alloc] init];
+  [registrar addMethodCallDelegate:instance channel:channel];
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    if (![FIRApp defaultApp]) {
+      [FIRApp configure];
+    }
+  }
+  return self;
+}
+
+- (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
+  if ([@"RemoteConfig#instance" isEqualToString:call.method]) {
+    FIRRemoteConfig *remoteConfig = [FIRRemoteConfig remoteConfig];
+    FIRRemoteConfigSettings *firRemoteConfigSettings = [remoteConfig configSettings];
+    NSMutableDictionary *resultDict = [[NSMutableDictionary alloc] init];
+
+    resultDict[@"LAST_FETCH_TIME"] = [[NSNumber alloc]
+        initWithLong:(long)[[remoteConfig lastFetchTime] timeIntervalSince1970] * 1000];
+    resultDict[@"LAST_FETCH_STATUS"] = [[NSNumber alloc]
+        initWithInt:[self mapLastFetchStatus:(FIRRemoteConfigFetchStatus)[remoteConfig
+                                                                              lastFetchStatus]]];
+    resultDict[@"IN_DEBUG_MODE"] =
+        [[NSNumber alloc] initWithBool:[firRemoteConfigSettings isDeveloperModeEnabled]];
+
+    result(resultDict);
```

Comment:
I would also put the current remote config values in here

Reply:
Done